lib/models/Meme.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+
+const memeSchema = new mongoose.Schema({
+
+  top: {
+    type: String,
+    required: false
+  },
+  image: {
+    type: String,
+    required: true,
+    default: 0,
```

Comment:
What do these default and min fields do for your image? Are they needed for a string type?